### PR TITLE
Receive work order

### PR DESCRIPTION
**Pull request by casperjureskog**
**PT Story:** https://www.pivotaltracker.com/story/show/144023775

**Changes proposed in this PR:**

* Adds index view show workorder
* Adds Foundation 
* Adds form for selecting status
* Add some CSSing

**What I have learned working on this feature:**
Use form_tag with select
Sett default value on select list dynamically

**Screenshots:**
screen shot 2017-04-24 at 11 42 56
screen shot 2017-04-24 at 11 42 13

### CODE REVIEW

File: app/controllers/workorders_controller.rb [hunk 1]
```diff
@@ -1,4 +1,17 @@
 class WorkordersController < ApplicationController
+  def index
+    @workorder = Workorder.all
```
Comment (AmberWilkie) [inline app/controllers/workorders_controller.rb]: If it's an array of things, we use a plural: `workorders`

File: app/controllers/workorders_controller.rb [hunk 2]
```diff
@@ -1,4 +1,17 @@
...
+  end
+
+  def update
+    @workorder = Workorder.find(params[:id])
+    if params[:status] != ''
```
Comment (AmberWilkie) [inline app/controllers/workorders_controller.rb:8]: Now that I think about it, I believe we can use `.blank?` See if that works the same way.

File: app/controllers/workorders_controller.rb [hunk 3]
```diff
@@ -8,6 +21,10 @@ def new
 
   def create
     @workorder = Workorder.new workorder_params
+    if params[:id]
+      @workorder.update(help_request_id: params[:id])
+      HelpRequest.find(params[:id]).update(workorder_id: @workorder.id)
```
Comment (AmberWilkie) [inline app/controllers/workorders_controller.rb]: This feels fishy. Isn't this `id` related to the workorder? How can it be used to find a `HelpRequest`? Make sure the fact that both ids are 1 in the test is not causing problems here.

If id does relate to the `HelpRequest`, we should probably go ahead and call it `help_request_id`

File: app/views/index/index.html.haml [hunk 4]
```diff
@@ -1,24 +1,49 @@
...
+              = work.status
+          -elsif work.status == 'Unable to complete'
+            .stat_red
+              = work.status
+          -elsif work.status == 'Next week'
+            .stat_blue
+              = work.status
+          -elsif work.status == 'Pending'
+            .stat_grey
+              = work.status
```
Comment (AmberWilkie) [inline app/views/index/index.html.haml]: All this if-this-status give-this-color business could be extracted to a helper method. You'd call the method, which would figure out what color (and return it), then you'd dynamically call the CSS class. It's not a blocker, but think about how that would clean this up.

**Review by AmberWilkie — COMMENTED** [review]
Ok a few small things and one big thing to think about.

File: app/views/index/index.html.haml (hunk 4 above)
Comment (casperjureskog) [reply app/views/index/index.html.haml]: yes, we vill do a shore for it in PT so we can fixit when we have down time between features

Comment (AmberWilkie) [reply app/views/index/index.html.haml]: ok fair enough. Write up a PT ticket please.

Comment (casperjureskog) [reply app/views/index/index.html.haml]: yes done

File: app/controllers/workorders_controller.rb (hunk 3 above)
Comment (Scharrre97) [reply app/controllers/workorders_controller.rb]: how?

File: app/views/workorders/new.html.haml [hunk 5]
```diff
@@ -20,7 +20,7 @@
     .actions
       = f.submit "Send"
 - else
-  = form_for @workorder, url: workorders_path(id: @help_requests.id),method: 'post' do |f|
+  = form_for @workorder, url: workorders_path(help_request_id: @help_requests.id),method: 'post' do |f|
```
Comment (AmberWilkie) [inline app/views/workorders/new.html.haml]: If this is one object, it should be called `@help_request` - no S. I expect `@help_requests` to be an array of requests, so you wouldn't be able to call `id` on it.

File: app/controllers/workorders_controller.rb (hunk 2 above)
Comment (Scharrre97) [reply app/controllers/workorders_controller.rb:8]: we try a second time but it doesn't work. I think we should keep it like this. thx

**Review by AmberWilkie — DISMISSED** [review]

**Review by AmberWilkie — APPROVED** [review]

**Pull request merged**